x/storage/keeper/keeper.go

```diff
@@ -2430,27 +2456,37 @@ func (k Keeper) UpdateObjectContent(
 		return types.ErrObjectIsUpdating.Wrapf("The object is already being updated")
 	}
 	// check permission
-	effect := k.VerifyObjectPermission(ctx, bucketInfo, objectInfo, operator, permtypes.ACTION_UPDATE_OBJECT_CONTENT)
+	var updater sdk.AccAddress
+	if opts.Delegated {
```

Comment:
can you compile this branch and run a node on testnet, so that we can verify that there is no breaking change introduced that cause consensus issue on chain.